### ISSUE TITLE
Permissions on downloaded files 

### ISSUE THREAD

**Issue by tyger** [abraunegg/onedrive#793]
Not sure whether it makes sense to put it here as a bug. Want to ask first and if not default behaviour will fill it according to requirements.
Haven't found anything in documentation. What are the permission `onedrive` supposed to create local files with? 
Right now downloaded files are created with `onedrive:users  rw-r--r--` where as services in run under `onedrive` user that has only one group `onedrive`.
So basically there is no way to edit file by anyone else.

**Comment by abraunegg** [comment]
@tyger 
The default behavior is to create files with the user & group of the UID / GID that is running the process.

For services, this is governed by the following in the systemd service files:
**onedrive.service:**
```
[Service]
ExecStart=/usr/local/bin/onedrive --monitor
Restart=on-failure
RestartSec=3
RestartPreventExitStatus=3
```

**onedriveAT.service:**
```
[Service]
ExecStart=/usr/local/bin/onedrive --monitor --confdir=/home/%i/.config/onedrive
User=%i
Group=users
Restart=on-failure
RestartSec=3
RestartPreventExitStatus=3
```

and the following for init.d:
**onedrive.init**
```
start() {
        export PATH=/usr/local/bin/:$PATH
        echo -n "Starting $APP_NAME: "
        daemon --user root onedrive_service.sh
        RETVAL=$?
        echo
        [ $RETVAL -eq 0 ] && touch /var/lock/subsys/onedrive || \
           RETVAL=1
        return $RETVAL
}
```

> So basically there is no way to edit file by anyone else

Generally, if you use the 'onedrive.service' file, this will usually run as 'root', whereas if you want the client to run as a normal user. you use 'onedriveAT.service' with the appropriate username added:

```
systemctl enable onedrive@<username>.service
systemctl start onedrive@<username>.service
```

Refer: https://github.com/abraunegg/onedrive/blob/master/docs/USAGE.md#running-onedrive-as-a-system-service

**Comment by abraunegg** [comment]
Closing issue as question answered
